compiler/src/lntoamm/UserFunction.ts

```diff
@@ -8,6 +8,8 @@ import StatementType from './StatementType'
 import Type from './Type'
 import { Args, Fn, } from './Function'
 import { LPNode, } from '../lp'
+import { stat } from 'fs'
+import { passCommandToAction } from 'commander'
```

Comment:
What are these imports for? They don't seem to be used.

Reply:
autocomplete :/